samples/Nimb3s.Automaton.Samples.CreateJobConsoleApp/Program.cs

```diff
@@ -12,8 +12,11 @@ static async Task Main()
 
             Console.Title = assemblyName;
 
+            await new NewsJobCreator().RunAsync();
+

```

Comment:
looks good 😃. a solid example is nice to have in the samples project.

Reply:
don't see the NewsJobCreator class, not sure if it's part of the commit?

Reply:
No, this was not part of the commit. I think this was added when we were testing it.